_dev/.stylelintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "ignoreFiles": [
+    "../packages/fxa-content-server/**"
```

Comment:
The payments server is also using `sass-lint`. I know we've listed content server in the `ignore` list here since there's a mountain of linter warnings there, but if we use this for payments as well how's that looking?

I also noticed the config standard doesn't lint against IDs? We probably want to set that moving forward with `fxa-settings`.

LGTM otherwise. :)

Reply:
fxa-payments has been updated and no IDs was added